### ISSUE TITLE
not well-formed (invalid token): line 315, column 12

### ISSUE THREAD

**Issue by targumanu** [deanishe/alfred-fixum#1]
When I try to do a dry run, I get a notification "not well-formed (invalid token): line 315, column 12".

Here's the workflow's log:

```bash
[2017-04-10 00:32:27][input.listfilter] Processing output of 'action.script' with arg 'dryrun'
[2017-04-10 00:32:28][ERROR: action.script] 00:32:27 workflow.py:2225 DEBUG    Workflow version : 0.1.2
00:32:27 fixum.py:186 INFO     ==================================================
00:32:27 fixum.py:187 DEBUG    opts={'--help': False,
 '--nothing': True,
 '--version': False}
00:32:27 fixum.py:188 INFO     looking for workflows using an outdated (buggy) version of Alfred-Workflow...
00:32:27 fixum.py:199 INFO     workflow directory: /Users/mike/Dropbox/App Data/Alfred.alfredpreferences/workflows
00:32:27 fixum.py:221 DEBUG    ignoring non-directory: .DS_Store
00:32:27 fixum.py:225 DEBUG    not an AW workflow: .DS_Store
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.00963676-91ED-4CB4-A028-24F5D16B4DAE
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.014366BF-3C70-408B-947E-49EE0E1EE2E0
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.0144444E-FF4C-4A58-9439-FDF5BDE6E983
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.01472C84-7AC5-4D1A-ADD9-9D5BB51B9B0A
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.014F7CA3-C8E5-4DD9-9270-9FF3BED07C04
00:32:27 fixum.py:225 DEBUG    not an AW workflow: user.workflow.015A9E2C-1A67-4E1F-A1CB-DD5C4DA1901A
00:32:28 fixum.py:225 DEBUG    not an AW workflow: user.workflow.01AA9B71-CC13-499A-8800-DF43A9419770
00:32:28 fixum.py:225 DEBUG    not an AW workflow: user.workflow.0340C6D2-FAA8-4196-994E-28F28D6F94E3
00:32:28 fixum.py:225 DEBUG    not an AW workflow: user.workflow.047CE1D3-5836-444F-9B45-C548077610AD
00:32:28 fixum.py:225 DEBUG    not an AW workflow: user.workflow.05308695-5C37-45D7-9265-14BD89F1A1A3
00:32:28 fixum.py:225 DEBUG    not an AW workflow: user.workflow.05D3104B-0F82-44DA-8695-4688554BBF22
00:32:28 workflow.py:2243 ERROR    not well-formed (invalid token): line 315, column 12
Traceback (most recent call last):
  File "/Users/mike/Dropbox/App Data/Alfred.alfredpreferences/workflows/user.workflow.31AD40AB-DA68-4E61-A6CE-DF07B7CC710D/workflow/workflow.py", line 2236, in run
    func(self)
  File "fixum.py", line 223, in main
    info = get_workflow_info(p)
  File "fixum.py", line 114, in get_workflow_info
    info = read_plist(ip)
  File "fixum.py", line 75, in read_plist
    return plistlib.readPlistFromString(xml)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plistlib.py", line 103, in readPlistFromString
    return readPlist(StringIO(data))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plistlib.py", line 78, in readPlist
    rootObject = p.parse(pathOrFile)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plistlib.py", line 406, in parse
    parser.ParseFile(fileobj)
ExpatError: not well-formed (invalid token): line 315, column 12
00:32:28 workflow.py:2246 INFO     For assistance, see: https://github.com/deanishe/alfred-fixum/issues
00:32:28 workflow.py:2267 DEBUG    Workflow finished in 1.017 seconds.
[2017-04-10 00:32:28][action.script] Processing output of 'output.notification' with arg 'not well-formed (invalid token): line 315, column 12'
```

**Comment by deanishe** [comment]
I can add code to catch the error, but the root problem is that you have an invalid workflow.

**Comment by deanishe** [comment]
Fixed in [v0.1.3](https://github.com/deanishe/alfred-fixum/releases/latest)
